File: Data Structure/3.Linked List/LinkedList To Array.java
// https://workat.tech/problem-solving/practice/linked-list-to-array
/** This is the ListNode class definition

class ListNode {
	int data;
	ListNode next;

	ListNode(int data) {
		this.data = data;
		this.next = null;
	}
}
**/

class Solution {
	List<Integer> linkedListToArray (ListNode head) {
	    // add your logic here
		ArrayList<Integer> l = new ArrayList<Integer>();
		
		ListNode currentNode = head;
		while(currentNode != null){
			l.add(currentNode.data);
			currentNode = currentNode.next;
		}
		
		return l;
	}
}

File: Data Structure/3.Linked List/LinkedListToArray.py
"""
This is the ListNode class definition

class ListNode:
	def __init__(self, data=0, next=None):
		self.data = data
		self.next = next

"""
# https://workat.tech/problem-solving/practice/linked-list-to-array

class Solution:
	def linkedListToArray(self, head: ListNode) -> List[int]:
		arr = []
		
		t  = head
		while(t!=None):
			arr.append(t.data)
			t = t.next
		return arr


